Dockerfile

```diff
@@ -1,22 +1,20 @@
 FROM  node:14.15-alpine as assets-builder
 
 RUN apk --no-cache add ca-certificates
+RUN apk --no-cache add make
```

Comment:
MInor thing I noticed but you might want to do to save of container layers:

```suggestion
RUN apk --no-cache add ca-certificates \
    apk --no-cache add make
```

Each instruction line like  `RUN`, `ADD` or `COPY` would create one distinct layer. On some platforms like [Heroku](https://devcenter.heroku.com/articles/container-registry-and-runtime#known-issues-and-limitations) there is a hard limit on the number of layers:

![image](https://user-images.githubusercontent.com/696529/107003929-1e05a400-67c0-11eb-805b-aa01b00df647.png)

Reply:
@olivierobert ✅  Fixed on [73d6280](https://github.com/carryall/go-google-scraper-challenge/pull/25/commits/73d62804c5db2804c93f12e64a033147a8154317)